containers/signing-service/install-signing-service.sh.j2

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+# Install signing-service dependencies
+/tmp/install-dependencies.sh
+
+virtualenv_dir=/usr/lib/virtualenv
+# Create virtualenv and Django app's dependencies
+virtualenv --python "python{{ python_version }}" "$virtualenv_dir"
+source "$virtualenv_dir/bin/activate"
+python -m pip install --upgrade pip
+python -m pip install --requirement "/usr/lib/signing-service/requirements.lock"
+(
+    cd /usr/lib/middleman_protocol/
+    python setup.py develop
+)
```

Comment:
As discussed before, Signing Service should have the relative path to `middleman_protocol` hard-coded in its dependencies. Then you will not have to install it explicitly.

Reply:
This solution must wait for [#736](https://github.com/golemfactory/concent/issues/736)